sphinx/cmdline.py

```diff
@@ -83,6 +84,19 @@ def handle_exception(app, args, exception, stderr=sys.stderr):
                   file=stderr)
 
 
+def jobs_argument(value):
+    # type (int, str) -> int
+    """
+    Special type to handle 'auto' flags passed to 'sphinx-build' via -j flag. Can
+    be expanded to handle other special scaling requests, such as setting job count
+    to (cpu_count / 2)
+    """
+    if value == 'auto':
```

Comment:
Supporting `-j 0` would be good too - that's a fairly standard way to indicate "use CPU count" in many applications.

Reply:
I don't know that. Could you tell me an example which uses `-j 0`?
I prefer to follow commonly used options.

Reply:
[Pylint](https://pylint.readthedocs.io/en/latest/user_guide/run.html#parallel-execution) supports `-j 0`, flake8 documents `-j auto` but at least as of the pervious version I was using they also support `-j 0` (they don't document this though).

For Sphinx, I'd follow the flake8 approach: support both, but document `-j auto`.